core/src/test/scala/org/apache/spark/util/UtilsSuite.scala

```diff
@@ -1310,6 +1313,18 @@ class UtilsSuite extends SparkFunSuite with ResetSystemProperties with Logging {
       assert(Utils.trimExceptCRLF(s"b${s}b") === s"b${s}b")
     }
   }
+
+  test("getSerializedLambda") {
+    // TODO: adds more tests
```

Comment:
are you adding more tests in this PR? If so, maybe remove and add a jira to add more tests?